go-auth-service/test_go_auth.sh

```diff
@@ -0,0 +1,190 @@
+#!/bin/bash
```

Comment:
Please use `env` so we can use these scripts on NixOS:
```suggestion
#!/usr/bin/env bash
```